renga_deployer/engines.py

```diff
@@ -96,6 +96,11 @@ def launch(self, execution, **kwargs):
         """Launch a docker container with the context image."""
         context = execution.context
 
+        execution.environment.update({
```

Comment:
Q: shouldn't this be `execution.environment.setdefault('DEPLOYER_BASE_URL', current_app.config.get('DEPLOYER_DEFAULT_BASE_URL'))`?

Reply:
A: I don't have a case in mind where the variable would have been set before and could be overwritten with something different here. So both options work for me.